src/docs/faq.md

```diff
@@ -0,0 +1,72 @@
+---
+id: faq
+title: FAQ
+---
+
+## What is Polywrap?
+
+Polywrap is a framework for porable and composable SDKs. Polywrap calls these SDKs “wraps”. Wraps are essentially WebAssembly-based modules.
+
+## What are wraps?
+
+Wraps are simply WebAssembly binaries built with Polywrap and packaged with specific metadata that allows clients to run them.
+
+## How are wraps run?
+
+Wraps are run by clients. Clients will fetch a wrap and execute it locally in the context of the application. Clients exist in a variety of languages: JavaScript, Rust, Python, etc.
+
+## How does this execution happen?
+
+In a nutshell, wrap execution follows these steps:
+
+- The application requests the execution of a wrap
+- Internally, the client:
+  - Fetches the requested wrap
+  - Executes the wrap's function inside a WebAssembly runtime
+- The client returns the result of the wrap's function call back to the application
+
+Or, if you prefer diagrams:
+
+![Client fetches wrap](/img/docs/faq/1-load.png)
+![Client executes wrap and returns execution result](/img/docs/faq/2-exec.png)
+## Where do wraps come from?
+
+Wraps can be bundled with the application, or loaded from external sources: file storage, the Internet, you name it. Clients allows developers to configure this.
+
+## Who/what can use wraps?
+
+Wraps are used within apps and other wraps. They are defined via their metadata, which allows apps to integrate wraps, and wraps to import other wraps.
+
+## Why wraps?
+
+- Wraps can run and live anywhere:
+  - Clients exist in [many languages](/clients).
+  - Wraps can be bundled with apps or downloaded when needed.
+- Wraps can call other wraps:
+  - Wrap metadata is imported at built-time.
+  - Wrap code is executed at run-time.
+- Wraps help secure your app:
+  - They execute in a secure WebAssembly environment.
+  - They don't have direct access to their host environment.
+
+## How do I access host capabilities?
+
+Wraps can import host functions defined by the application developer. These native functions allow wraps to access host capabilities. Polywrap packages these capabilities as “plugins”.
```

Comment:
```suggestion
Polywrap has a plugin system that enables application developers to allow access to host capabilities. Wraps can call host functions defined in plugins. Plugins are also infused with Polywrap superpowers--bindings that allow them to appear to the Polywrap client and be invoked just like a wrap.

Unlike wraps, plugins are not portable. Plugins are written in a Polywrap Client's native language, and must be registered with the client at runtime.
```

Reply:
I added more to this one as well. If you like what I'm doing, let me know and I can look at the other questions more closely as well.